res/css/views/rooms/_RoomHeader.pcss

```diff
@@ -112,7 +112,7 @@ limitations under the License.
     }
 
     &[aria-expanded="true"] {
-        background-color: $quinary-content;
+        background-color: var(--cpd-color-gray-400);
```

Comment:
This will break custom themes e.g. https://github.com/aaronraimist/element-themes